Entity Explorer - Linux Host.ipynb

```diff
@@ -136,7 +136,7 @@
    "source": [
```

Comment:
<p>I moved these from the extra_imports since it's less obscure and more succinct to have them "classic-style"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>